sql/core/src/test/scala/org/apache/spark/sql/execution/streaming/state/RocksDBStateStoreSuite.scala

```diff
@@ -918,6 +918,297 @@ class RocksDBStateStoreSuite extends StateStoreSuiteBase[RocksDBStateStoreProvid
     }
   }
 
+  /* Column family related tests */
```

Comment:
Are the tests added here copied from RocksDBSuite? If it is, it'd be a great help for reviewer if you go through and comment whenever there is a difference worth looking at.

Reply:
Added few comments below to denote which suites are removed entirely and which are moved into `RocksDBStateStoreSuite`.